simtools/configuration/commandline_parser.py

```diff
@@ -410,6 +413,14 @@ def azimuth_angle(angle):
         except ValueError:
             logger.debug(
                 "The azimuth angle provided is not a valid numeric value. "
+                "Will check if it is an astropy.Quantity instead"
+            )
+        try:
+            fangle = u.Quantity(angle).to("deg")
+            return fangle.to("deg")
+        except TypeError:
```

Comment:
There are (at least) two other exceptions that can be raised here. One is the case where the user provides a valid quantity but it cannot be converted to deg (e.g., kg), and the other is when the unit provided is not even a valid unit (e.g., "90 north").
Technically an error will be raised that will provide some details. However, perhaps you want to provide an even friendlier error message (far from critical though).

Reply:
I do think the error message `astropy.units.core.UnitConversionError: 'kg' (mass) and 'deg' (angle) are not convertible` is very clear and hard to improve. Also think that `ValueError: 'north' did not parse as unit` is quite clear and difficult to improve. Suggest not too add additional exceptions.